apps/src/util/thumbnail.js

```diff
@@ -154,7 +154,9 @@ export function captureThumbnailFromElement(element) {
   }).catch(e => {
     console.log(`error capturing screenshot: ${e}`);
   }).then(() => {
-    isCapturePending = false;
+    setTimeout(() => {
+      isCapturePending = false;
+    }, 100);
```

Comment:
setTimeout inside a promise chain feels a little like an antipattern.  What exactly are we waiting for here?  @davidsbailey  might be able to suggest an alternative.

Reply:
I think I have a better solution — see commit 3d38c6749def8a2fe4447742e9e251c2149b5786.